EnRonSwanson/BangazonCLI
# Rock get customer list

## Fixes 
# Fleshes out the GetListCustomer method on the Customer Manager.

## Changes proposed in this pull request: 
- Made the GetListCustomer method get a list of customers
-
-

## How to test: 
- !!!Must have AddCustomer Method method merged before proper test of this method.
### Test 1
- Pull branch, dotnet run program
- Add multiple customers
- Choose option 2 for picking active customer
- should show list of customers
### Test 2
- Go to CustomerManagerShould Test file
- Run GetListCustomerShould test
- Profit

[Ticket] (https://github.com/EnRonSwanson/BangazonCLI/issues/31)
@EnRonSwanson/
